src/benchmark/file_io_read_micro_benchmark.cpp

```diff
@@ -14,7 +14,9 @@
 
 namespace hyrise {
 
-void read_data_using_read(const size_t from, const size_t to, int32_t fd, uint32_t* read_data_start) {
+void read_data_using_read(const size_t from, const size_t to, int32_t fd, uint32_t* read_data_start, bool& threads_ready_to_executed) {
+  while(!threads_ready_to_executed){}
```

Comment:
I think I would call this `threads_ready_to_be_executed`

Reply:
or maybe `threads_executing` or `threads_execution`

Reply:
Will do :)